### PR TITLE
Added matches

### PR DESCRIPTION
**Pull request by Tamas-Leung**
image

### CODE REVIEW

File: src/backend/src/services/ws/index.js [hunk 1]
```diff
@@ -145,6 +145,7 @@ export default class WebSocketManager {
 
       const won = client.lastCompletedRound === game.round;
       client.ws.send(JSON.stringify({ method: Events.END, endData: { won } }));
+      insertMatch(gameID, clientId, won, game.problemsPlayed);
```
Comment (YoussefRizkalla) [inline src/backend/src/services/ws/index.js:148]: Missing await

File: src/backend/src/services/ws/index.js [hunk 2]
```diff
@@ -167,6 +168,7 @@ export default class WebSocketManager {
         );
         client.game = null;
         client.lastCompletedRound = 0;
+        insertMatch(gameID, clientId, false, game.problemsPlayed);
```
Comment (YoussefRizkalla) [inline src/backend/src/services/ws/index.js:171]: Missing await

File: src/backend/src/services/ws/index.js (hunk 1 above)
Comment (Tamas-Leung) [reply src/backend/src/services/ws/index.js:148]: doesn't need await cause we don't need to wait for it

**Review by ZhimingZhao — APPROVED** [review]

**Review by AntonKanug — APPROVED** [review]

**Comment by AntonKanug** [comment]
Closes #33

**Comment by AntonKanug** [comment]
Closes #104

**Pull request merged**